src/GPEmulator.jl

```diff
@@ -64,28 +64,49 @@ struct GPObj{FT<:AbstractFloat, GPM}
     models::Vector
     "whether to fit GP models on normalized inputs ((inputs - input_mean) * sqrt_inv_input_cov)"
     normalized::Bool
+    "the singular value decomposition matrix"
+    decomposition
+    "the normalization in the svd transformed space"
+    sqrt_singular_values
```

Comment:
Can we add the types of these attributes?

Reply:
Done.